src/WebJobs.Script/Content/Script/res.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+module.exports = () => {
+    var res = {};
```

Comment:
@mathewc brought that up as well so I think we're on the same page and could move forward with the rename, unless if you feel strongly that it should remain `res`.